### DISCUSSION TITLE
RowSelector for CTkTable

### DISCUSSION THREAD

**Discussion by CallMePixelMan** [Akascape/CTkTable#34] [Show and tell]
Hello, if any of you need something that lets you select one or more rows from a CTkTable (with CTRL key support), I made a wrapper around the CTkTable in order to get the values easely for any project, with styling support.

Here's a quick demo with two tables, and two different CTkTableRowSelector configuration (blue lines are those that are selected).
![image](https://github.com/Akascape/CTkTable/assets/65821351/5c1a95ca-7d3e-4667-9c81-64d2e5f9f37f)

The API is really simple, I will soon add some documentation and automated test with GitHub action (100% test coverage btw).

You can install it using `pip install ctktablerowselector`, going to the [PyPi webpage](https://pypi.org/project/ctktablerowselector/) or take a look to the [GitHub Repo](https://github.com/CallMePixelMan/CTkTableRowSelector).

Hope you like it, thanks. ;)

**Comment by Akascape** [comment]
@CallMePixelMan Nice, but I have also added select row/column method recently

**Reply by CallMePixelMan** [reply]
With CTRL key support ?

**Reply by Akascape** [reply]
Not that binding added. But it can be added manually.

**Reply by CallMePixelMan** [reply]
If you're referring to select_row and select_column, this is not the same, this is a wrapper around CTkTable that lets users click on the row to select the entire row, instead of the cell, and get all the row easily with styling.

**Reply by Jackkkkk494** [reply]
Can you upload an example of how this method works please! Have got it to select specific cells and output values just not the entire row.

**Reply by Akascape** [reply]
@Jackkkkk494 Refer this: https://github.com/Akascape/CTkTable/issues/29
